packages/graphback-runtime-mongodb/tests/MongoDataProviderTest.ts

```diff
@@ -37,54 +34,25 @@ describe('MongoDBDataProvider Basic CRUD', () => {
     }
   ]
 
-
-  //Create a new database before each tests so that
-  //all tests can run parallel
-
-
-  // eslint-disable-next-line @typescript-eslint/tslint/config
-  afterEach(async (done) => {
-    if (context) {
-      await context.server.stop();
-    }
-
-    done();
-  })
-
-  test('Test missing "_id: GraphbackObjectID" primary key', async () => {
-    const schema = `
-    """
-    @model
-    """
-    type MissingPrimaryKeyModel {
-    id: ID!
-    text: String
-    }
-    `;
-
-
-    const defautConfig = {
-      "create": true,
-      "update": true,
-      "findOne": true,
-      "find": true,
-      "delete": true,
-      "subCreate": true,
-      "subUpdate": true,
-      "subDelete": true
-    }
-
-    try {
-      const metadata = new GraphbackCoreMetadata({ crudMethods: defautConfig }, buildSchema(schema));
-      const models = metadata.getModelDefinitions()
-      for (const model of models) {
-        new MongoDBDataProvider(model, undefined);
-      }
-      expect(true).toBeFalsy(); // should not reach here
-    } catch (error) {
-      expect(error.message).toEqual('Model "MissingPrimaryKeyModel" must contain a "_id: GraphbackObjectID" primary key. Visit https://graphback.dev/docs/model/datamodel#mongodb to see how to set up one for your MongoDB model.');
-    }
-  });
+  afterEach(async () => context?.server?.stop());
+
+  // test('Test missing "_id: GraphbackObjectID" primary key', async () => {
+  //   const schema = `
+  //   """
+  //   @model
+  //   """
+  //   type MissingPrimaryKeyModel {
+  //     id: ID!
+  //     text: String
+  //   }
+  //   `;
+
+  //   try {
+  //     context = await createTestingContext(schema);
+  //   } catch (error) {
+  //     expect(error.message).toEqual('Model "MissingPrimaryKeyModel" must contain a "_id: GraphbackObjectID" primary key. Visit https://graphback.dev/docs/model/datamodel#mongodb to see how to set up one for your MongoDB model.');
+  //   }
+  // });
```

Comment:
Why is this commented out?

Reply:
Sorry, I was trying to figure out why the tests were hanging.

Reply:
:-) No worries. This is good to go.